Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -6464,8 +6464,8 @@ public void Actions_InteractiveRebinding_CanBeRestrictedToSpecificControlPaths()
             Assert.That(rebind.candidates, Has.Count.EqualTo(4));
             Assert.That(rebind.candidates, Has.Exactly(1).SameAs(keyboard.spaceKey));
             Assert.That(rebind.candidates, Has.Exactly(1).SameAs(mouse.leftButton));
-            Assert.That(rebind.candidates[2], Is.SameAs(mouse.button));
-            Assert.That(rebind.candidates[3], Is.SameAs(keyboard.anyKey)); // Last place for AnyKey.
+            Assert.That(rebind.candidates[3], Is.SameAs(mouse.button));
+            Assert.That(rebind.candidates[2], Is.SameAs(keyboard.anyKey)); // Last place for AnyKey.
```

Comment:
Note: turning AnyKeyControl into a ButtonControl flipped the order here. I think that should be fine, as long as space key comes first?

Reply:
Yup, should be fine. Neither of the two synthetic buttons seems a better fit than the other.